server/channels/store/sqlstore/user_store.go

```diff
@@ -2140,7 +2140,11 @@ func (us SqlUserStore) IsEmpty(excludeBots bool) (bool, error) {
 		From("Users")
 
 	if excludeBots {
-		builder = builder.LeftJoin("Bots ON Users.Id = Bots.UserId").Where("Bots.UserId IS NULL")
+		if isPostgreSQL := us.DriverName() == model.DatabaseDriverPostgres; isPostgreSQL {
```

Comment:
A bit uncommon to use a variable again as the result of the expression. I'd have just used `if us.DriverName() == model.DatabaseDriverPostgres {`. But not a big deal.

Reply:
Trying new things :) It's generally easy to read this way but I can remove if you think it's doing harm than good 0/5

Reply:
Not against this, but it also made me read it twice, I thought I had missed something the first time.

Reply:
I think this style is valuable when the condition is too cumbersome to understand (maybe too long, or the variables involved are not self-documenting). But in this specific case, I think it's clear that we're checking that the driver is PostgreSQL